css-box-model/css-box-model-quiz-notes.md

```diff
@@ -0,0 +1,39 @@
+# css-box-model-quiz-notes
+
+## Quiz Questions
+
+After completing this exercise, you should be able to discuss or answer the following questions:
+
+- What CSS properties make up the box model?
+
+- Which CSS property pushes boxes away from each other?
+
+- Which CSS property add space between a box's content and its border?
+
```

Comment:
Don't forget the quiz questions!

Reply:
My apologies, new one should have pushed through.